Sources/ShortcutUI/SwiftUI/NavigationLink+Button.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  NavigationLink+Button.swift
+//  ShortcutUI
+//
+//  Created by Swathi on 2022-06-09.
+//  Copyright © 2021 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+public struct NavigationLinkButton<Destination: View, Label: View>: View {
+    var action: () -> Void
+    var destination: () -> Destination
+    var label: () -> Label
+
+    @State private var isActive: Bool = false
```

Comment:
Always rely on type inference

Reply:
Fixed. My mistake I am so used to this style of coding. Will keep track